osf/models/preprint_service.py

```diff
@@ -43,6 +46,8 @@ class PreprintService(DirtyFieldsMixin, GuidMixin, BaseModel):
     # Format: [[root_subject._id, ..., child_subject._id], ...]
     subjects = DateTimeAwareJSONField(default=list, null=True, blank=True)
 
+    identifiers = GenericRelation(Identifier, related_query_name='preprintservices')
```

Comment:
Just one place -- in the script to add identifiers to preprints that don't have them -- https://github.com/CenterForOpenScience/osf.io/pull/7133/files#diff-12a91336af7651551db5ec264ee2eee2R14